traveler-ios-ui/TravelerKitUI/Views/DateRangeCell.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  DateRangeCell.swift
+//  TravelerKitUI
+//
+//  Created by Rakin Hoque on 2019-11-22.
+//  Copyright © 2019 GuestLogix. All rights reserved.
+//
+
+import UIKit
+
+protocol DateRangeCellDelegate: class {
+    func dateRangeCellDidUpdate(_ cell: DateRangeCell, startDate: Date)
+    func dateRangeCellDidUpdate(_ cell: DateRangeCell, endDate: Date)
+}
+
+open class DateRangeCell: UITableViewCell {
+    @IBOutlet weak var startDateField: UITextField!
+    @IBOutlet weak var endDateField: UITextField!
+    
+    weak var delegate: DateRangeCellDelegate?
+    
+    public var startDate: Date? {
```

Comment:
views should not be model aware. Views should be dumb, this logic does not belong here. It should be abstracted into a view controller and the cell should only care about receiving strings to put in its text fields.